### PR TITLE
add docs for creating, update and deleting cluster queues with REST API

### PR DESCRIPTION
**Pull request by angusfretwell**
Add docs for https://github.com/buildkite/buildkite/pull/12471

### CODE REVIEW

**Review by jordandcarter — COMMENTED** [review]
Looks good! I'll let @mbelton-buildkite give it the tick though

File: pages/apis/rest_api/clusters.md [hunk 1]
```diff
@@ -346,6 +346,138 @@ Required scope: `read_clusters`
 
 Success response: `200 OK`
 
+### Create a queue
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/queues" \
+  -H "Content-Type: application/json" \
+  -d '{ "key": "default", "description": "The default queue for this cluster" }'
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/clusters.md:354]: I thought a cluster already had a default queue when created? If so, perhaps this should create a queue with a different key and description to be more realistic?

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me! I've added one minor comment to consider, but happy for you to decide how to proceed with it. 😊

File: pages/apis/rest_api/clusters.md (hunk 1 above)
Comment (angusfretwell) [reply pages/apis/rest_api/clusters.md:354]: They do when created through the UI however we decided to change this behaviour for the API so it's more predictable and the user is in full control!

**Pull request merged**